# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/test/webgl-conf/moz.yaml
# Commit: 8e0aea33f622
# Full Hash: 8e0aea33f622a76c5c8baa7dcf4485ffefd1662f
# Author: Kelsey Gilbert <kelsey.gilbert@mozilla.com>
# Date: 2024-07-09 09:45:28
# Regressor Bug: 1892542
# File Overlap Count: 1
# Description:
#   Bug 1892542 - Webgl e.g. createBuffer() made infallible. r=gfx-reviewers,webidl,saschanaz,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D208050
# ==============================================================================

diff -r b953e0ccb68b -r 8e0aea33f622 dom/canvas/test/webgl-conf/moz.yaml
--- a/dom/canvas/test/webgl-conf/moz.yaml	Mon Jul 08 19:36:12 2024 +0300
+++ b/dom/canvas/test/webgl-conf/moz.yaml	Mon Jul 08 17:24:12 2024 +0000
@@ -13,8 +13,8 @@
 
   license: MIT
 
-  release: commit 4c5b8bfe586d983fae6a0571cc702f43e5f5b719 Tue Apr 2 14:17:01 2024 -0700
-  revision: 4c5b8bfe586d983fae6a0571cc702f43e5f5b719
+  release: commit cd04892d7d7ac986a83383b06bfa792c9c369f8c Tue Apr 2 14:43:28 2024 -0700
+  revision: cd04892d7d7ac986a83383b06bfa792c9c369f8c
 
 
 updatebot: